src/linear_algebra/basic.lean

```diff
@@ -302,6 +302,20 @@ rfl
 theorem comp_smul (g : M₂ →ₗ[R] M₃) (a : R) : g.comp (a • f) = a • (g.comp f) :=
 ext $ assume b, by rw [comp_apply, smul_apply, g.map_smul]; refl
 
+/--
+The family of linear maps `M₂ → M` parameterised by `f ∈ M₂ → R`, `x ∈ M`, is linear in `f`, `x`.
+-/
+def smul_rightₗ : (M₂ →ₗ[R] R) →ₗ[R] M →ₗ[R] M₂ →ₗ[R] M := {
```

Comment:
The braces should move to the next line, so that you get
```lean
{ to_fun := ...
```

Reply:
Thanks!

One of these days, I'll stop thinking I'm writing JavaScript 🤦‍♂